src/backend/marsha/core/storage.py

```diff
@@ -1,15 +1,30 @@
 """Customizing Django storage backends to enable blue/green deployments."""
+from collections import OrderedDict
+import re
+
 from django.conf import settings
 from django.contrib.staticfiles.storage import ManifestFilesMixin, StaticFilesStorage
 
 from storages.backends.s3boto3 import S3Boto3Storage
 
 
+STATIC_NOHASH_REGEX = re.compile(getattr(settings, "STATIC_NOHASH_PATTERN", "^$"))
+
+
 class ConfigurableManifestFilesMixin(ManifestFilesMixin):
     """Allow configuring the name of the manifest file via Django settings."""
 
     manifest_name = settings.STATICFILES_MANIFEST_NAME
 
+    def post_process(self, paths, dry_run=False, **options):
+        """Remove paths from file to post process."""
```

Comment:
Some js static files generated by webpack already have a unique name `per build` and may be referenced from within the js applications. We therefore don't want to hash their name and include them in the manifest file.
We use a regex configurable via settings to decide which files to ignore.